core/templates/dev/head/pages/exploration_editor/editor_tab/exploration_editor_tab.html

```diff
@@ -0,0 +1,56 @@
+<div ng-controller="ExplorationEditorTab" class="oppia-editor-cards-container">
+  <div class="oppia-editor-header">
+    <state-name-editor></state-name-editor>
+  </div>
+
+  <div ng-if="areParametersEnabled()">
```

Comment:
Shouldn't this and the following be replaced with something like `<state-editor></state-editor>`?